modules/Kconfig.tfm

```diff
@@ -76,17 +76,20 @@ config TFM_ISOLATION_LEVEL
 	  1,2 or 3; the default is set by build configuration.
 
 choice TFM_BL2
-        prompt "BL2 configuration, should TFM build with MCUboot support"
-        default TFM_BL2_CONFIG_FILE_DEFAULT
+	prompt "BL2 configuration, should TFM build with MCUboot support"
```

Comment:
Could you revert the white space diffs here, @agansari ?
Or are they actually "fixes"?

Reply:
multiple occasions in the file

Reply:
@ioannisg yes, it's a fix, there were different indentations in the file.
Do you see a problem with the new indentation?